### PR TITLE
feat: nibi verified asset + update explorer link

### PR DESCRIPTION
**Pull request by cgilbe27**
## Description

Adding chain: Nibiru

### Upgrading Asset to Verified

If upgrading an Asset to Verified, please see the requirements specified at [LISTING](https://github.com/osmosis-labs/assetlists/blob/main/LISTING.md#upgrade-asset-to-verified-status-permissioned), and ensure the following:
- [x] Asset is defined thoroughly at the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - [x] A meaningful `description` (and `extended_description`, unless classified as 'meme').
   - [x] Associated `socials`, including `website` and `twitter`.
   - [x] Logo Image has a square Aspect Ratio, < 250 KB file size, and appropriate visual contrast with Osmosis Zone colors.
- [x] This pool contains contains at least $1k USD-worth of liquidity of the asset (Provide Pool ID): 1620

'Verified' Status Validation Checklist (to be completed by Osmosis Zone maintainers):
- [x] Verify appearance and metadata
- [x] Accurate Price
- [x] Trading and routing functionality
- [x] Withdraw and Deposit
   - [x] Deposit Transaction URL (if in-app)

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -4445,9 +4028,15 @@
       "listing_date_time_utc": "2024-05-29T10:27:00Z",
       "osmosis_unlisted": true,
       "_comment": "Alloyed USDT $allUSDT",
-      "categories": [
-        "stablecoin"
-      ]
+      "categories": ["stablecoin"]
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:4033]: Please leave a `comment` with the name and ticker symbol

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -4445,9 +4028,15 @@
       "listing_date_time_utc": "2024-05-29T10:27:00Z",
       "osmosis_unlisted": true,
       "_comment": "Alloyed USDT $allUSDT",
-      "categories": [
-        "stablecoin"
-      ]
+      "categories": ["stablecoin"]
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:4031]: Please do not reformat the entire file

**Comment by cgilbe27** [comment]
Related MR: https://github.com/cosmos/chain-registry/pull/4802 @JeremyParish69

**Comment by JeremyParish69** [comment]
> Should I set verified to true?

Not yet, that would be another PR. You would remove the former paragraphs of the PR Description Template, but keep the Upgrade to Verified Asset section, and tick the checklist on that PR.

File: osmosis-1/osmosis.zone_assets.json [hunk 3]
```diff
@@ -4456,6 +4456,16 @@
           "withdraw_url": "https://app.picasso.network/?from=OSMOSIS&to=SOLANA"
         }
       ]
+    },
+    {
+      "chain_name": "nibiru",
+      "base_denom": "unibi",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:4462]: Please add the `path` property

**Comment by JeremyParish69** [comment]
Hold on, [$NIBI has already be listed](https://github.com/osmosis-labs/assetlists/blob/87deaf2860c1e796cae88b7336279e788617d030/osmosis-1/osmosis.zone_assets.json#L3451-L3461). What is the purpose of this PR?

**Comment by cgilbe27** [comment]
Apologies, I was unaware this was already done, I did need to update the explorer MR

We are trying to get our asset to be a verified asset @JeremyParish69

File: osmosis-1/osmosis.zone_assets.json [hunk 4]
```diff
@@ -4445,7 +4445,7 @@
       "chain_name": "composable",
       "base_denom": "ibc/9D5DA3720001F91DD76B8F609A93F96688EC8185B54BF9A1A1450EB34FF2D912",
       "path": "transfer/channel-1279/transfer/channel-71/ATeTQcUkWGs7AZ15mCiFUWCW9EUL7KpDZEHCN1Y8pump",
-      "osmosis_verified": false,
+      "osmosis_verified": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:4448]: This property is for the the wrong asset ($WHINE), but I think you mean for it to be $NIBI

**Pull request merged**